### DISCUSSION TITLE
Focus next / prev instead of focus left or right?

### DISCUSSION THREAD

**Discussion by b11e** [nikitabobko/AeroSpace#785] [questions-and-answers]
Hi people, 

I often have my windows stacked behind each other fullscreen and I forget whether I want to move left or right.

Given
`
h_tiles
├── window 1 (focused)
└── window 2
`
Move focus right
`
h_tiles
├── window 1
└── window 2 (focused)
`
Move focus right or left doesn't matter with 2 windows
`
h_tiles
├── window 1 (focused)
└── window 2
`

Is there a way to achieve this? I'd like to focus next or prev without it being strict on direction. Sort of rotating the windows like a carousel.

**Comment by whoop-t** [comment]
I do this to get it to wrap when moving right

```
alt-tab = 'focus right --boundaries-action wrap-around-the-workspace'
```

**Reply by b11e** [reply]
perfect, thank you.

`cmd-h = 'focus left --boundaries-action wrap-around-the-workspace'`
`cmd-l = 'focus right --boundaries-action wrap-around-the-workspace'`
